indexer-app/src/cmr/indexer/data/concepts/tag.clj

```diff
@@ -15,15 +15,14 @@
       {:concept-id concept-id
        :deleted deleted}
       {:concept-id concept-id
-       :tag-key.lowercase (str/lower-case tag-key)
+       :tag-key.lowercase (util/safe-lowercase tag-key)
```

Comment:
Not entirely sold on why we have to make these changes. It is not possible to have a nil tag-key and when this happens, I would rather see an exception with the indexing failed rather than our code silently ignores the problem and index bad stuff in elasticsearch.

Reply:
This is a good point ...

Reply:
I got your point. It crossed my mind too.  However, my thinking is that if something that can not be nil becomes nil, it will fail in many places rather than just the indexing...  Also, seems that when we added these lower-case ones, we didn't really know clearly if the field could be nil, so there could be many cases that it would cause trouble.  So I thought unless we have a clear list on which fields could be nil, which I don't have that list, maybe it's better to use safe-lowercase. Just my thought.